src/Illuminate/Database/Eloquent/Attributes/CollectedBy.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Illuminate\Database\Eloquent\Attributes;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+class CollectedBy
+{
+    /**
+     * Create a new attribute instance.
+     *
+     * @param  class-string  $collectionClass
```

Comment:
this should probably match the typehint in [Database\Eloquent\Model#L228](https://github.com/laravel/framework/blob/d62e92d781327bc2d272724f9f4be9dd62d5bceb/src/Illuminate/Database/Eloquent/Model.php#L228)

`@param  class-string<\Illuminate\Database\Eloquent\Collection<*, *>>  $collectionClass`